data-browser/src/components/forms/hooks/useSaveResource.ts

```diff
@@ -29,6 +29,10 @@ export const useSaveResource = (
       setErr(null);
 
       try {
+        if (resource.new) {
```

Comment:
Should maybe be moved to after the save as we probably don't want to notify when the save fails.